### DISCUSSION TITLE
Won't download

### DISCUSSION THREAD

**Discussion by icecreams71** [audiamus/BookLibConnect#106] [Q&A]
Hi,
I have use this software with great satisfaction with my Italian account and now I wanted to switch to my UK one.
Adding the new library was a breeze. Then I go on to sync, all good. I add my books for download and everything gets stuck.
The books are listed correctly for download and the purple arrow is shown. 
"Run" and "Abort" are greyed out and nothing is happening.
Any ideas?
Thanks in advance!

**Comment by audiamus** [comment]
Are the books selected?  And, most important: Does the download folder exist?

**Reply by suneecat** [reply]
I have the same issue in the USA.
I just installed the software. It shows the title but it will not download.

**Reply by audiamus** [reply]
Same question: If the book is in the main window, is selected, and the Run button is still disabled: Do the download and export folders exist?

**Reply by RafaelCaroQuintero** [reply]
![image](https://github.com/audiamus/BookLibConnect/assets/90371474/d8ef2317-8d3a-4e4a-9735-b69bad0a7e79)
I have wishlisted some books but can't see those in download tab or can u guide how to download file in audio format

**Comment by audiamus** [comment]
It appears you have no titles at all here. Have you created your profile for the correct Amazon region / marketplace? The log file and associated json files will tell, whether the query was successful.

**Comment by SchnuSchnu** [comment]
I have the same problem with 3 profiles that I have used successfully before. Now the run and abort button (as shown in the picture) is greyed out with all three profiles.
The Download folder does exist.
![Screenshot 2024-04-13 152744](https://github.com/audiamus/BookLibConnect/assets/162493403/35e98f5a-f666-4ce3-a531-8d7c38042153)

**Reply by SchnuSchnu** [reply]
SOLVED: I just saw that the export folder did not exist! So after creating it it now works! Thanks for the help!
